packages/algorand-js/test/teal-files/escrow.teal

```diff
@@ -0,0 +1,25 @@
+#pragma version 2
+// Check if Transaction type is pay, Receiver is john, fee <= 10000 AND amount is <= 100
```

Comment:
Please update the comment as per order of the checks, ex: type is pay, amount <= 100...

Reply:
alright